### DISCUSSION TITLE
D235 Video doorbell cannot add to HA

### DISCUSSION THREAD

**Discussion by phardy2410** [JurajNyiri/HomeAssistant-Tapo-Control#794] [Discuss]
As part of the HA integration I have to have a camera account. But Tapo in their infinite wisdom only allow a camera account to be added in "always in" mode aka wired not battery mode. This is to save the battery obviously. But now I can't add the D235 to HA.

Is there a way to add it without a Camera Account? Or bypass the always on requirement to allow a camera account to be added to the doorbell?

Fully appreciate this could well be a Tapo restriction over an integration issue but all support appreciated.

**Comment by JurajNyiri** [comment]
Duplicate of https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/discussions/808
